src/bulma.tsx

```diff
@@ -259,15 +245,12 @@ export function removeLoadingProps(props: Bulma.Loading) {
     return rest;
 }
 
-export function getColorModifiers({ isColor: color }: Bulma.Color) {
```

Comment:
leave `{ isColor: color }`. Here we use interface from Typescript for type-check, it means that `Bulma.Color` is an object that has `isColor` property and we are just destructing in a `color` variable.

Reply:
So would it make sense to name the property in the interface just `color` instead of `isColor`? Since it's just a field and not a function.. is that right?

Reply:
Well... That would makes sense... but this prop `isColor` is the one se use informe Bloomer and this change would break previous versions.

isColor was the name because Bulma use `is-white`. Si this sounds similar.

Does that makes sense?

Reply:
Ah, I see. I was thinking of it more as "is-X" like a function that returns boolean, but that makes sense.

Reply:
Yes, a little confusing since we had a `isColor` function that you removed. Didn't saw that coming, that was really bad naming from my side.

Reply:
^ especially confusing for someone who doesn't know the type system.. I was totally sure they were related some way for a while. But yeah, hard to see coming.

Reply:
Even for me, I totally forgot about it, that's why I had other reviews comments to delete that, hahaha.